themes/_core/js/checkout-payment.js

```diff
@@ -25,6 +25,15 @@
 import $ from 'jquery';
 import prestashop from 'prestashop';
 
+prestashop.checkout = prestashop.checkout || {};
+
+// eslint-disable-next-line no-unused-vars
+prestashop.checkout.onCheckOrderableCartError = (resp, paymentObject) => {
```

Comment:
```suggestion
prestashop.checkout.onCheckOrderableCartError = (resp) => {
```
What's the reason to keep the paymentObject var?

Reply:
I think @NeOMakinG added it so payment modules can intercept this depending on payment method selected?